client/command_data/skillorder.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @fileoverview Command Data & Constants Related to the /about command
+ * These constants are used to dynamically generate a slash command on discord.
+ * This File also contains the constants values used exclusively in the /about command.
+ * @author Ai0796
+ */
+
+module.exports = {
+    'INFO': {
+        'name': 'skillorder',
+        'utilization': '/skillorder',
+        'description': 'Show the current T100 leaderboard',
```

Comment:
You might have overlooked this, but the description is still the same as another command.